auth/src/main/java/com/firebase/ui/auth/util/data/ProviderUtils.java

```diff
@@ -99,7 +102,7 @@ public static String signInMethodToProviderId(@NonNull String method) {
             case EmailAuthProvider.EMAIL_LINK_SIGN_IN_METHOD:
                 return EMAIL_LINK_PROVIDER;
             default:
-                throw new IllegalStateException("Unknown method: " + method);
+                return method;
```

Comment:
Yes, the user can have sign in methods that we don't support. Does throwing an unchecked exception provide any value though?
It looks like this is used in exactly one place, and if it is an unsupported method it will simply not be in the allowed providers list. I don't really think crashing is the better option.